config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib $(QT5_DIR)/bin
+  lit_libs += Qt5Core
+
+  verbatim (make, macros) {
+    CPPFLAGS += -fPIC
+  }
+
+  verbatim (gnuace, macros) {
+    CPPFLAGS += -fPIC
+  }
+
+  specific (prop:windows) {
+    // On Windows it's possible for Qt to include winsock.h through windows.h
+    // which will conflict with winsock2.h. This forces the preprocessor to skip
+    // winsock.h.
+    macros += _WINSOCKAPI_
+  }
+
+  Define_Custom(UIC) {
```

Comment:
In MPC's Qt4 projects, these rules belong to the GUI library mpb, not the core one.

Reply:
That's true but `moc` and `qrc` could be used in non-gui applications, that's the reason I moved them here.